QUARKUS-1325.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-1325 - Helm chart support for Quarkus
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1325
+
+This test plan cover Quarkus - Helm integration
+
+## Scope of the testing
+
+Verify a repeatable/versioned way to deploy Quarkus application into K8s/ocp through helm
+
+Test development will focus on
+- Generate and deploy a Quarkus Helm Chart over Openshift default repository
+- Verify some mapping between Quarkus configuration and generated Helm chart:
+  - Quarkus SmallRye Health 
+
```

Comment:
Will you ensure both JVM and Native applications get deployed?

Reply:
That is a good question... I think that JVM would be enough because, from Helm side point of view, they don't care about native or JVM, Helm will install a Chart that will point to a docker container registry that has the app image (Native or JVM)

So... I think that we should only cover JVM mode because from my point of view native mode is more expensive in terms of resources and doesn't add any value to this scenario.

WDYT?

Reply:
From testing perspective it seems to be fine to focus just on JVM mode.

We should probably consider what's the flow for users, do they provide just image in one mode or they do both `abc-jvm` and `abc-native` images. This is not exclusively tight to helm topic, it's more generic topic for pushing the images from Quarkus based project. My view is that usually one mode gets picked and that' it. Dual image push may not be the common case.

But pls comment about this in TP to indicate this topic was considered + add the reasoning.
`For this first coverage we don't think that native mode is relevant.` is too generic, please indicate that Helm is concerned just about image name and doesn't care if the app is JVM or Native based.